cpp/rocksdb4pl.cpp

```diff
@@ -1400,8 +1393,7 @@ static bool
 enum_key_prefix(const enum_state& state)
 { if ( state.type == ENUM_PREFIX )
   { rocksdb::Slice k(state.it->key());
-    return ( k.size_ >= state.prefix.length() &&
-	     memcmp(k.data_, state.prefix.data(), state.prefix.length()) == 0 );
+    return k.starts_with(state.prefix);
```

Comment:
I think that this code is equivalent but please check against
https://github.com/facebook/rocksdb/blob/981fd432fa2441fc10a59a462bd14906ccb1c0e0/include/rocksdb/slice.h#L112